Backtracking/generateParentheses.js

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Should be in a proper JSDoc comment.

Reply:
Hh Sorry It's the first time I see what JSDoc comments are actually xD

Reply:
Please refer to https://jsdoc.app/about-getting-started.html - you should be using `@` annotations.

Reply:
Done 👍